int16 id
float32 x
float32 y
int16 value
string name